syntax = "proto3";

service PlateDetection {
    rpc DetectPlate (ImageRequest) returns (DetectionResponse);
}

message ImageRequest {
    bytes image = 1;  // Envia a imagem como bytes
}

message DetectionResponse {
    string plate_text = 1;
    int32 x1 = 2;
    int32 y1 = 3;
    int32 x2 = 4;
    int32 y2 = 5;
}
